### PR TITLE
Added required env vars and templates for the build analyser cli

### PR DESCRIPTION
**Pull request by harshad16**
- Added required environment variable for thoth-common and openshift to interact with build-analyser.
- OpenShift Templates for the build analysers is added.
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: openshift/job-build-analyse-template.yaml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+  - name: THOTH_BUILD_LOG_DOC_ID
+    description: Document id of Build log present in ceph.
+    displayName: Build Log Document ID
+    required: true
+
+  - name: THOTH_REPORT_OUTPUT
+    description: Remote where results should be send to.
+    displayName: ImageStream Tag
+    required: false
```
Comment (fridex) [inline openshift/job-build-analyse-template.yaml]: Is this always required in the deployment (result-api?)? Here and bellow as well.

File: openshift/job-build-dependencies-template.yaml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-bucket-name
+                      name: thoth
+                - name: THOTH_CEPH_BUCKET_PREFIX
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-bucket-prefix
+                      name: thoth
+                - name: THOTH_USER_API_APP_SECRET_KEY
```
Comment (fridex) [inline openshift/job-build-dependencies-template.yaml]: I think this env is specific to user-api. It is not needed in these yaml templates.

File: openshift/job-build-analyse-template.yaml (hunk 1 above)
Comment (harshad16) [reply openshift/job-build-analyse-template.yaml]: yes, this should have been required, I forgot to change it after testing.

File: openshift/job-build-dependencies-template.yaml (hunk 2 above)
Comment (harshad16) [reply openshift/job-build-dependencies-template.yaml]: copy-paste mistake, how can I get those :eyes:?

**Comment by harshad16** [comment]
@fridex  Have a look at these as well, if any changes are needed let me know :)

**Review by fridex — APPROVED** [review]
Thanks! :+1: :hatched_chick:

**Pull request merged**